python/hail/methods/statgen.py

```diff
@@ -1309,9 +1310,32 @@ def pc_relate(call_expr, min_individual_maf, *, k=None, scores_expr=None,
     col_keys = hl.literal(mt.col_key.collect(), dtype=tarray(mt.col_key.dtype))
     return ht.key_by(i=col_keys[ht.i], j=col_keys[ht.j])
 
-class SplitMulti(object):
+
+@typecheck(ds=oneof(Table, MatrixTable),
+           keep_star=bool,
+           left_aligned=bool)
+def split_multi(ds, keep_star=False, left_aligned=False) -> Union[Table, MatrixTable]:
     """Split multiallelic variants.
 
+    The resulting dataset will be keyed by the split locus and alleles.
```

Comment:
These docs are mostly (almost entirely) copied from the `SplitMulti` class and the `split_multi_hts` docs.